cmd/resim/commands/batch.go

```diff
@@ -80,7 +82,21 @@ func createBatch(ccmd *cobra.Command, args []string) {
 		log.Fatal(err)
 	}
 	experienceIDs := parseUUIDs(viper.GetString(experienceIDsKey))
-	experienceTagIDs := parseUUIDs(viper.GetString(experienceTagIDsKey))
+
+	if viper.GetString(experienceTagIDsKey) != "" && viper.GetString(experienceTagNamesKey) != "" {
```

Comment:
This checks if both are present; do you also need to check that at least one is present?

Also, consider `viper.IsSet(key)` for this (and below).  (We should probably use that in more places).

Reply:
> do you also need to check that at least one is present?

we need to check that at least one of experienceIDs or experienceTagID/Name is present. I had thought Simon had a PR to do this, but I think it was actually just a comment. I can do manually.